src/ps/scripts.ps1

```diff
@@ -1,76 +1,67 @@
 function GetLatestSlackVersionFolder {
-    $versions = 
-        { [int]($_.Name -replace '.*?-(\d+)\.(\d+)\.(\d+)', '$1') }, 
-        { [int]($_.Name -replace '.*?-(\d+)\.(\d+)\.(\d+)', '$2') }, 
-        { [int]($_.Name -replace '.*?-(\d+)\.(\d+)\.(\d+)', '$3') }
-
-    $latestVersionFolder = Get-ChildItem $env:LOCALAPPDATA\slack\app-* | Sort-Object $versions -Descending | Select-Object -First 1 -ExpandProperty Name
+    $latestVersionFolder = $filename = cmd /c "dir $env:LOCALAPPDATA\slack\app* /ad /on /b" | select -last 1
```

Comment:
Regarding the cmd dir - Nice! My goal is to use Powershell Core so it can be run on all OSes but I haven't got full support for Mac or Linux yet. As long as we can detect OS, this should be ok for a Windows-specific case if it really gives any speed benefit. It's looking like Slack might clean up old versions when it installs a new version so probably will only have two app version folders at a time.

Regarding the removal of the versions object sort - this doesn't work. Add a folder called app-3.4.10 and you'll see it won't be selected. Need to make some unit tests soon for this.

Reply:
I did notice that in testing that it wouldn't sort 3.4.10, but they never use double digits on their versioning so I didn't see that as an issue. 
That does seem like a very extensive article on quotes. simply though, just use single on plain strings. 

Measure-Command is a great cmdlet to help you optimize for speed if you ever find that necessary.